packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -0,0 +1,492 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
```

Comment:
If possible, please don't import `dart:io`, otherwise this won't be marked as a "web-compatible" plugin.